lib/config/defaults.js

```diff
@@ -709,14 +709,14 @@ const applyModuleDefaults = (
 		}
 		if (css) {
 			const cssRule = {
-				type: CSS_MODULE_TYPE,
+				type: CSS_MODULE_TYPE_AUTO,
 				resolve: {
 					fullySpecified: true,
 					preferRelative: true
```

Comment:
@alexander-akait would you know we prefer relative paths for normal css files but not for css modules?

Reply:
I think it is a bug